### PR TITLE
New sections api endpoints, image upload endpoints, bug fixes

### PR DESCRIPTION
**Pull request by markjeffries**
Add profile fields and ability to skip onboarding

Added new section api calls

Fix delete endpoints returning a json decode error

Added calls for uploading images

### CODE REVIEW

File: service/users.json [hunk 1]
```diff
@@ -277,8 +277,19 @@
                     "location": "json", 
                     "required": true, 
                     "type": "string"
+                },
+                "entryFlowComplete": {
+                    "default": 0,
+                    "description": "Set to 1 to bypass BaseKit onboarding process",
+                    "enum": [
+                        "0",
+                        "1"
```
Comment (robjmills) [inline service/users.json:286]: these are strings vs the default which is an int - does this matter?

**Review by robjmills — APPROVED** [review]
Looks good to me @markjeffries  - one tiny question 🚢

File: service/users.json (hunk 1 above)
Comment (markjeffries) [reply service/users.json:286]: Good question, I wonder if the enum has to be strings

Comment (markjeffries) [reply service/users.json:286]: In my example script I'm passing through a numeric so I guess not

**Pull request merged**